campaign/controllers.py

```diff
@@ -2867,3 +2867,43 @@ def update_campaignx_from_politician(campaignx, politician):
         'save_changes': save_changes,
     }
     return results
+
+
+def delete_campaign_supporter(voter_to_delete=None):
+    
+    status = ""
+    success = True
+    campaign_supporter_deleted = 0
+    campaign_supporter_not_deleted = 0
+    voter_to_delete_id = voter_to_delete.id
```

Comment:
Hello @hesamshayegan, this should be voter_to_delete.we_vote_id instead of ".id"